0000-Proportional-Service-Provider Rewards

```diff
@@ -0,0 +1,63 @@
+# HIP XX: Proportional Service Provider Rewards
+- Author(s): @zer0tweets, @meowshka (Nova Labs, Inc.), @KeithRettig (Discord @KeithR)
+- Start Date: 2023-05-29
+- Category: Economic, Technical
+- Original HIP PR: #nnn
+- Tracking Issue: #nnn
+- Voting Requirements: veMOBILE Holders
+
+## Summary
+This HIP proposes to make modifications to how Service Providers are eligible for rewards from the Service Provider MOBILE Reward bucket.
```

Comment:
This summary is a bit vague and lacks the core points of your argument:

Can you summarize the whole text in this part? Be sure to include the main points. Should look something like this and you can use this as a starting point:

The proposal suggests using a usage-based approach to calculate rewards for Service Providers in the Helium MOBILE Network, based on the amount of Data Credits burned. Service Providers would be rewarded in MOBILE tokens at a 1:1 ratio for the burned Data Credits, with proportional distribution if the total exceeds the reward bucket value.